### PR TITLE
implement config.sh for inside iso

### PR DESCRIPTION
**Pull request by bastelfreak**
this updates the scripts and documentation about the usage of config.sh.
It will be copied into our ISO directory, the customize_airootfs.sh
script will source it and use the variables if available.

this commit also updates the .bash_profile to run the .automated_script.sh.

### CODE REVIEW

**Comment by bastelfreak** [comment]
During build I currently get:

``` bash
/root/customize_airootfs.sh: line 44: ISO_NFSSERV: unbound variable
```

File: airootfs/root/customize_airootfs.sh [hunk 1]
```diff
@@ -34,14 +40,21 @@ pacman -Rdd --noconfirm openresolv netctl dhcpcd
 # populate trust information
 trust extract-compat
 
+# add installimage NFS mounts
+if [ -n "$ISO_NFSSERV" ]; then
+  echo "${ISO_NFSSERV}:/srv/nfs/installimage /root/.installimage nfs ro 0 0" >> /etc/fstab
+  echo "${ISO_NFSSERV}:/srv/nfs/images /root/images nfs ro 0 0" >> /etc/fstab
+  mkdir -p /root/images
+  mkdir -p /root/.installimage
```
Comment (killermoehre) [inline airootfs/root/customize_airootfs.sh]: Can be merged with the line above to `mkdir -p /root/{images,.installimage}`

File: airootfs/root/customize_airootfs.sh [hunk 2]
```diff
@@ -34,14 +40,21 @@ pacman -Rdd --noconfirm openresolv netctl dhcpcd
...
+fi
+
 # update package information
 pacman -Syy
 pkgfile --update
+
+# remove config, could contain sensitive information and we don't want to ship it in the ISO
+rm config.sh
```
Comment (killermoehre) [inline airootfs/root/customize_airootfs.sh]: What happens if this fails for some reason?

File: airootfs/root/.bash_profile [hunk 3]
```diff
@@ -1 +1,2 @@
+. ~/.automated_script.sh
```
Comment (killermoehre) [inline airootfs/root/.bash_profile:1]: always check for file existance before. this is not python where you can ask for forgivness. this is truly bloody bash

File: airootfs/root/customize_airootfs.sh [hunk 4]
```diff
@@ -4,8 +4,12 @@
 # modified by bastelfreak
 # based on work from Bluewind and foxxx0
 ##
+
 set -e -u
 
+# get our config
+. /root/config.sh
```
Comment (killermoehre) [inline airootfs/root/customize_airootfs.sh:19]: check for existance

File: airootfs/root/customize_airootfs.sh (hunk 2 above)
Comment (bastelfreak) [inline airootfs/root/customize_airootfs.sh]: implemented a check for the existence of config.sh

**Comment by bastelfreak** [comment]
ACK from @foxxx0 on IRC

**Pull request merged**